tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -564,7 +564,7 @@ func storeLimitCommandFunc(cmd *cobra.Command, args []string) {
 		if args[0] == "all" {
 			prefix = storesLimitPrefix
 			if rate > maxStoreLimit {
-				cmd.Printf("rate should less than %f for all\n", maxStoreLimit)
+				cmd.Printf("rate should less than %v for all\n", int(maxStoreLimit))
```

Comment:
We support float for setting the store limit.

Reply:
How about `rate should less than 200.0 for all`?

Reply:
LGTM